airflow/www/app.py

```diff
@@ -70,8 +71,27 @@ def create_app(config=None, testing=False, app_name="Airflow"):
     flask_app = Flask(__name__)
     flask_app.secret_key = conf.get('webserver', 'SECRET_KEY')
 
-    session_lifetime_days = conf.getint('webserver', 'SESSION_LIFETIME_DAYS', fallback=30)
-    flask_app.config['PERMANENT_SESSION_LIFETIME'] = timedelta(days=session_lifetime_days)
+    default_session_lifetime = 43200
+    if conf.has_option('webserver', 'SESSION_LIFETIME_MINUTES'):
+        session_lifetime_minutes = conf.getint(
+            'webserver', 'SESSION_LIFETIME_MINUTES', fallback=default_session_lifetime
+        )
+    else:
+        if conf.has_option('webserver', 'SESSION_LIFETIME_DAYS') or conf.has_option(
+            'webserver', 'FORCE_LOG_OUT_AFTER'
+        ):
+            logging.error(
+                '`SESSION_LIFETIME_DAYS` option from `webserver` section has been '
+                'renamed to `SESSION_LIFETIME_MINUTES`. New option allows to configure '
+                'session lifetime in minutes. FORCE_LOG_OUT_AFTER option has been removed '
+                'from `webserver` section. Please update your configuration.'
+            )
+            stop_webserver()
+        else:
+            logging.info('Using default value for `SESSION_LIFETIME_MINUTES`: %s', default_session_lifetime)
+            session_lifetime_minutes = default_session_lifetime
```

Comment:
This `if-else` condition check is allowing one situation which should not happen in my opinion: if a user has both `SESSION_LIFETIME_MINUTES` AND `SESSION_LIFETIME_DAYS`/`FORCE_LOG_OUT_AFTER` in his/her configuration, NO error will be raised. Of course it will correctly use `SESSION_LIFETIME_MINUTES`. However, when another user looks at this configuration, it will be confusing and misleading.

So I would like to suggest change the `if-else` check into structure below:

```python
if conf has SESSION_LIFETIME_DAYS or FORCE_LOG_OUT_AFTER:
    logging.error() + exit()

if conf.has_option('webserver', 'SESSION_LIFETIME_MINUTES'):
    session_lifetime_minutes = conf.getint('webserver', 'SESSION_LIFETIME_MINUTES', ......)
else:
    logging.info("Using default value for SESSION_LIFETIME_MINUTES)
    session_lifetime_minutes = default_session_lifetime
```

Let me know your thoughts? Thanks.

Reply:
This is how it looked in the first version of PR. 😉 Later with @mik-laj we came to [conclusion ](https://github.com/apache/airflow/pull/11745#discussion_r510427359) that we should not penalized users when `SESSION_LIFETIME_MINUTES` is passed. It's hard to say which approach is better.  
After all I think it will be better to stop execution when removed options are provided because it will enforce users to update confiuration and it will be less confusing.

Reply:
Maybe @mik-laj shares if you have different opinion please.

Reply:
@mik-laj Could you share your thoughts ?

Reply:
I agree. We can raise an exception when the configuration contains options that we no longer support.

Reply:
@XD-DENG I've just updated the code.